src/action/payment.js

```diff
@@ -87,7 +87,11 @@ class PaymentAction {
       return this._notification.display({ msg: 'Enter an invoice or address' });
     }
     if (await this.decodeInvoice({ invoice: this._store.payment.address })) {
-      this._nav.goPayLightningConfirm();
+      this._store.syncedToChain
+        ? this._nav.goPayLightningConfirm()
+        : this._notification.display({
+            msg: 'Unable to make Lightning payment when syncing to chain',
+          });
```

Comment:
It would probably be more readable to put `if (!this._store.syncedToChain)` in its own block and display the notification like in the address check above.

Reply:
Just to clarify... are you suggesting to put the `if (!this._store.syncedToChain)` above the `if (await this.decodeInvoice({ invoice: this._store.payment.address }))`? If that's the case then we'd also be restricting on-chain bitcoin payments until synced.

Alternatively we could replace the ternary operator with:
```
if (this._store.syncedToChain) {
  this._nav.goPayLightningConfirm();
} else {
  this._notification.display({
    msg: 'Unable to make Lightning payment when syncing to chain',
  });
}
```
which doesn't read much better 🤔